src/shared/actions/example.ts

```diff
@@ -0,0 +1,27 @@
+import {createAliasedAction} from "electron-redux";
+
+export const EXAMPLE = 'EXAMPLE';
+
+export function exampleAction(text: string) {
+  return {
+    type: 'EXAMPLE',
+    payload: {text}
+  }
+}
+
+export function exampleLocalAction(text: string) {
+  return {
+    type: 'EXAMPLE',
+    payload: {text},
+    meta: {
+      scope: 'local'
+    }
+  }
+}
+
+export const exampleAliasedAction = createAliasedAction(
+  'EXAMPLE', (text: string) => ({
+    type: 'EXAMPLE',
+    payload: {text}
+  })
+)
```

Comment:
Notice three different notations for actions.
1. received in both processes
2. `renderer` -> `renderer`, used for e.g. UI state like `isPanelOpen`
3. `main` -> `renderer`